projects/ccf-rui/src/app/shared/components/organ-selector/organ-selector.component.theme.scss

```diff
@@ -8,7 +8,7 @@
   .carousel {
```

Comment:
Again this is a completely unrelated component. Why are you changing it in this pr?